controllers/apartment_controller.rb

```diff
@@ -0,0 +1,49 @@
+# INDEX
+get '/apartments/' do
+  # binding.pry
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+# NEW
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+# CREATE
+post '/apartments' do
+  # get new artist name from params entered from form
+  @new_apartment = params[:new_apartment]
+  # create new artist
+  Apartment.create(address: @new_apartment)
+  # redirect to artist index
+  redirect '/apartments'
+end
+
+# SHOW
+get '/apartments/:id' do
+  @apartment = Apartment.find(params[:id])
+  erb :"apartments/show"
+end
+
+# EDIT
+get '/apartments/:id/edit' do
+  @artist = Apartment.find(params[:id])
+  erb :"apartments/edit"
+end
+
+# UPDATE
+put '/apartments/:id' do
+  @edited_apartment = params[:apartment]
```

Comment:
I'd call this `@apartment_params`... although, honestly, you don't really even need a variable for this, since you're not using it in your views.